assets/scss/05-dataviz/_datatable.scss

```diff
@@ -15,4 +15,24 @@
       text-align: #{$option};
     }
   }
+  
+  .rt-tr {
+    transition: background 3.2s ease;
+    
+    &:hover .rt-td {
+      background: rgba(0,0,0,0.07);
```

Comment:
I wonder if this should be opaque and use one of the color variables, e.g. $c-gray-light;